plugins/inputs/sqlserver/azuresqldbqueries.go

```diff
@@ -233,21 +233,24 @@ SELECT
 		WHEN slo.[edition] = 'Hyperscale' then NULL
 		ELSE CAST(DATABASEPROPERTYEX(DB_NAME(),'MaxSizeInBytes') as bigint)/(1024*1024)
 	END AS [total_storage_mb]
-	,CASE
+	,(SELECT SUM(CAST(FILEPROPERTY(name, 'SpaceUsed') AS INT) / 128) FROM sys.database_files WHERE type_desc = 'ROWS') AS used_space_data_mb
```

Comment:
```suggestion
	,(SELECT SUM(CAST(FILEPROPERTY(name, 'SpaceUsed') AS INT) / 128) FROM sys.database_files WHERE type_desc = 'ROWS') AS used_storage_mb
```
It would be better to mention "data" explicitly, but an existing column name `available_storage_mb` doesn't and we cannot change it to avoid a breaking change, so my suggestion is to opt for consistency.

Reply:
I'm going to go with consistency as well to avoid questions or confusion as to why one says one thing and the other says something else.